### ISSUE TITLE
1 make a login screen 

### ISSUE THREAD

**Issue by cykeltur** [UbaidurRehman1/ChartF#8]

**Comment by UbaidurRehman1** [comment]
Will start now, and will give you first prototype in next 3 days

**Comment by cykeltur** [comment]
Good the first one is the easiest one :-)

**Comment by UbaidurRehman1** [comment]
yup starting from easiest one, step by step, so all will be clear

**Comment by cykeltur** [comment]
Don't overdo revision 1 of this giu part 🤣

**Comment by cykeltur** [comment]
The "sparkle" can come later...
Save the time for:
The "flowchart wizardpart"..

**Comment by cykeltur** [comment]
example:

[step1_loginexample.zip](https://github.com/UbaidurRehman1/ChartF/files/3390366/step1_loginexample.zip)

**Comment by cykeltur** [comment]
Please see the 2f user data #21
Comments if database integration starting now...
